src/Components/Breakdown.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { Promise } from 'mongoose';
+class Breakdown extends Component {
+    constructor (){
+        super()
+        this.state={
+            foodSum:0,
+            entertainmentSum:0,
+            salarySum:0
+        }
+        
+    }
+
+    feltirData (){
+       
+        let ob={}
+        this.props.state.users.forEach(e => {ob[e.category]?ob[e.category]+=e.amount:ob[e.category]=e.amount });
+            
+       const arry=Object.entries(ob)
```

Comment:
You don't need `Object.entries` here. Return the object and use `Object.keys` in the `render` method.